### DISCUSSION TITLE
article/oblog/kai-pian-wo-de-2015

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#7] [Announcements]
# article/oblog/kai-pian-wo-de-2015

“有些事情不是看到希望才去坚持，而是坚持了才会看到希望”。一直很喜欢这句话，它也成了我每次失败之后安慰自己的最大法宝，这也使得我在技术学习的道路上越陷越深…… 在过去的2015年，充实、忙碌、收获满满。 年初的时候，在同学的邀请下，接触了一个非常不错的团队，受团队成员对工作热情的感染以及能在技术团队管理和架构上可以完全掌控的诱惑下加入了，也因为这个决定几乎透支了上半年的所有夜晚和周末。从技术选型到系统架构，利用敏捷管理和版本控制推进开发过程，部署自动化测试、构建、交付系统减少人力成本，最后到上云方案和运维，几乎将过去所有积累的各类经验筛选后一一付诸实践，并获得了不错的效果，在团队发展越来越好的同时，对技术团队的任务要求也就越来越大，时间上的矛盾也愈发激烈，但是在谈及对技术团队的维持和价值理解上存在太大的落差（可参考最近知乎上创始人与技术合伙人的撕逼大战），带着对技术劳动的敬畏和对付出劳动的技术人们的抱歉不得不离开。虽然各种遗憾，但在这半年的各种尝试中，也验证了自己的技术架构和管理上的一些思路的好与坏，对于初出茅庐的我来说这次的经历“得”远大于“失”！

https://www.didispace.com/article/oblog/kai-pian-wo-de-2015.html

**Comment by osanben** [comment]
白猫咨询打不开了为啥

**Reply by dyc87112** [reply]
之前国服关闭了，后来没继续玩，就没维护了
